terminus-ui/chip/src/chip-collection.component.ts

```diff
@@ -357,34 +346,43 @@ export class TsChipCollectionComponent implements AfterViewInit, OnInit, OnDestr
   @Output()
   public readonly tabUpdateFocus = new EventEmitter();
 
-  constructor(protected elementRef: ElementRef<HTMLElement>,
-              private changeDetectorRef: ChangeDetectorRef,
-              public zone: NgZone,
-              @Optional() private dir: Directionality,) {
+
+  constructor(
+    protected elementRef: ElementRef<HTMLElement>,
+    private changeDetectorRef: ChangeDetectorRef,
+    public zone: NgZone,
+  ) {
     zone.runOutsideAngular(() => {
       this._role = this.empty ? null : 'listbox';
     });
   }
 
-  public ngOnInit() {
+
+  /**
+   * Initialize the selection model
+   */
+  public ngOnInit(): void {
     this.selectionModel = new SelectionModel<TsChipComponent>(this.allowMultipleSelections, undefined, false);
   }
 
-  public ngAfterViewInit() {
+
+  /**
+   * Initialize the key manager and listen for chip changes
+   */
+  public ngAfterViewInit(): void {
     this.keyManager = new FocusKeyManager<TsChipComponent>(this.chips)
       .withWrap()
       .withVerticalOrientation()
-      .withHorizontalOrientation(this.dir ? this.dir.value : 'ltr');
+      .withHorizontalOrientation('ltr');
```

Comment:
I ended up just removing directional support since we don't support that elsewhere. If we ever get into a `rtl` situation we'll need to address the library as a whole.

Reply:
sounds good 👍